docs/rancher/node/rke2-cluster.md

```diff
@@ -43,9 +42,24 @@ Users can create a RKE2 Kubernetes cluster from the **Cluster Management** page
 8. Enter **Image** (required).
 9. Enter **Network Name** (required).
 10. Enter **SSH User** (required).
-11. Click **Create**.
+11. Configure the **Show Advanced > User Data** to enable the VMs to install required packages.
+```yaml
+#cloud-config
+packages:
+  - iptables
```

Comment:
```suggestion
packages:
  - iptables
  - qemu-guest-agent
```

Reply:
We already injected qemu-guest-agent in node driver logic.

They will be merged.